seqr/views/apis/family_api.py

```diff
@@ -294,8 +300,13 @@ def update_family_pedigree_image(request, family_guid):
 
     update_model_from_json(family, {'pedigree_image': pedigree_image}, request.user)
 
+    if isinstance(family.pedigree_image, ImageFieldFile):
+        try:
+            updated_image = family.pedigree_image.url
+        except Exception:
+            updated_image = None
```

Comment:
no

Reply:
The `_get_json_for_family` behavior is to keep the `result.pedigreeImage` rather than replace it with `url` if the field is not an `ImageFieldFile` instance. The new code will generate an undefined `pedigreeImage` in the same scenario.